src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -126,7 +129,8 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 					bw = new BufferedWriter(new FileWriter(outfilename + "-orig.xml"));
 					bw.write(inxml);
 				} catch (IOException e2) {
-					logger.error(e.getMessage(), e);
+					logger.error(Constant.messages.getString("report.write.diskspace.error.log.message"), e);
+					JOptionPane.showMessageDialog(View.getSingleton().getMainFrame(), Constant.messages.getString("report.write.diskspace.error.log.message"));

```

Comment:
(Also, the function should return, no point trying to continue with the I/O operations if the previous failed.)